README.md

```diff
@@ -1,6 +1,394 @@
 # CNS Object Broker
-## For dynmically provisioning object storage compliant with an S3 interface.
 
 ## WARNING!
+The work in this project is a proof of concept and not intended for production.
 
-The work in this project is a proof of concept and not intended for anything more than that.
+## Utilize Kubernetes Service-Catalog to dynamically provision CNS Object Storage.
+
+## Overview
```

Comment:
How about this intro?

A core feature of Kubernetes storage is the ability to dynamically provision **block** and **file** storage on demand. The CNS Object Store project demonstrates that by using the Kubernetes-Incubator's Service-Catalog, it is now possible to dynamically provision S3 compatible **object** storage. This is accomplished using [s3-gluster-kubernetes](https://github.com/jarrpa/gluster-kubernetes), [service-catalog](https://github.com/kubernetes-incubator/service-catalog), and a custom written [CNS object broker](https://github.com/copejon/cns-object-broker) to deploy the object service.

The CNS Object Broker handles requests to create and destroy object buckets and returns information required to consume them. Our [command flow diagram]() shows some of the resources created and where Kubernetes, Service-Catalog, and the (often external) service broker interact. Service-catalog terms used below are included in this diagram.

Reply:
I like it.  Instead of the last stanza, I've moved the "Topology" section to immediately follow the 2nd paragraph here.